manifests/stage-whereabouts-cni/0060-ip-reconciler-job.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 NVIDIA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: ip-reconciler
+  namespace: {{ .RuntimeSpec.Namespace }}
+  labels:
+    tier: node
+    app: whereabouts
+spec:
+  concurrencyPolicy: Forbid
+  successfulJobsHistoryLimit: 0
+  schedule: "*/5 * * * *"
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+      template:
+        metadata:
+          labels:
+            app: whereabouts
+        spec:
+          priorityClassName: "system-node-critical"
+          serviceAccountName: whereabouts
+          containers:
+            - name: whereabouts
+              image: {{ .CrSpec.Repository }}/{{ .CrSpec.Image }}:{{ .CrSpec.Version }}
+              resources:
+                requests:
+                  cpu: "100m"
+                  memory: "50Mi"
+              command:
+                - /ip-reconciler
+                - -log-level=verbose
```

Comment:
Do we really need verbose log level here?

Reply:
changed to `info`